openmc/model/model.py

```diff
@@ -132,14 +134,14 @@ def _materials_by_id(self):
 
     @property
     @lru_cache(maxsize=None)
-    def _cells_by_id(self):
+    def _cells_by_id(self) -> dict:
         """Dictionary mapping cell ID --> cell"""
         cells = self.geometry.get_all_cells()
         return {cell.id: cell for cell in cells.values()}
 
     @property
     @lru_cache(maxsize=None)
-    def _cells_by_name(self):
+    def _cells_by_name(self) -> dict:
```

Comment:
You can use typing.Dict to make this more specific
```suggestion
    def _cells_by_name(self) -> Dict[int, openmc.Cell]:
```